core/modules/css/testKvInterfaceImplMySql.cc

```diff
@@ -111,7 +111,7 @@ struct TestDBGuard {
         // need config without database name
         MySqlConfig sqlConfigLocal = sqlConfig;
         sqlConfigLocal.dbName = "";
-        std::cout << "config:" << sqlConfigLocal.asString() << std::endl;
+        std::cout << "config:" << sqlConfigLocal.toString() << std::endl;
```

Comment:
`toString()` is not needed here, you have stream insertion operator for that object.